sdk/js/src/token_bridge/redeem.ts

```diff
@@ -59,6 +60,8 @@ export async function redeemOnTerra(
   });
 }
 
+export const redeemOnInjective = createWrappedOnInjective;
```

Comment:
This code is somewhat confusing. It so happens that all of these functions are implemented the same way (just by submitting the VAA), so instead we should have a function like `submitVAAInjective` and implement
```ts
export const redeemOnInjective = submitVAAInjective
```

Reply:
Fixed